cni/pkg/nodeagent/net.go

```diff
@@ -231,8 +231,8 @@ func (s *NetServer) RemovePodFromMesh(ctx context.Context, pod *corev1.Pod) erro
 
 	openNetns := s.currentPodSnapshot.Take(string(pod.UID))
 	if openNetns == nil {
-		log.Warn("failed to find pod netns")
-		return fmt.Errorf("failed to find pod netns")
+		log.Warn("failed to find pod netns during removal")
```

Comment:
Wdyt about including the pod UID or pod name in the log? Could that be useful for debugging purposes?

Reply:
We are using `WithLabels` on the logger with the podname a few lines above this, so we should have that, at least.